### PR TITLE
DRILL-4511: Add unit test for "Table does not exist" situation in case of empty directory or incorrect table

### PR DESCRIPTION
**Pull request by KulykRoman**
- Added unit test for "Table does not exist" situation in case of empty directory or incorrect table.

### CODE REVIEW

File: exec/java-exec/src/test/java/org/apache/drill/exec/store/parquet/TestParquetMetadataCache.java [hunk 1]
```diff
@@ -207,6 +207,31 @@ public void testAbsentPluginOrWorkspaceError() throws Exception {
         .go();
   }
 
+  @Test //DRILL-4511
+  public void testTableDoesNotExistError() throws Exception {
```
Comment (arina-ielchiieva) [inline exec/java-exec/src/test/java/org/apache/drill/exec/store/parquet/TestParquetMetadataCache.java]: 1. Please split into two tests.
2. You can use to `ExecTest.getTempDir()` to get unique temporary directory.

**Comment by KulykRoman** [comment]
Done.

**Comment by arina-ielchiieva** [comment]
+1
